feature/platform/transceiver/zr_tunable_parameters_test/README.md

```diff
@@ -1,15 +1,25 @@
-# TRANSCEIVER-5: Configuration: 400ZR channel frequency and output TX launch power setting. 
+# TRANSCEIVER-5: Configuration: 400ZR channel frequency, output TX launch power and operational mode setting. 
 
 ## Summary
 
-Validate setting 400ZR tunable parameters channel frequency and output TX
-launch power and verify corresponding telemetry values.
+Validate setting 400ZR tunable parameters channel frequency, output TX
+launch power and operationla mode and verify corresponding telemetry values.
 
 ### Goals
 * Verify full C band frequency tunability for 100GHz line system grid.
 * Verify full C band frequency tunability for 75GHz line system grid.
 * Verify adjustable range of transmit output power across -13 to -9 dBm
-  in steps of 1 dB.  
+  in steps of 1 dB.
+* Verify  that the ZR module Host Interface ID and Media Interface ID
+  combination to ZR module AppSel mapping can be configured through the OC
+  Operatioanl-Mode. operational-mode is a construct in OpenConfig that masks
```

Comment:
In order to change the operational mode, I think we have to provide guidance for what the mode that should be changed is.    A test could get the list of available operational modes and switch from mode 'a' to mode 'b' to prove it can be configured.   But I am not sure if this is quite that simple?

Are these the operational-modes you are referring to:  [/terminal-device/operational-modes/mode](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-terminal-device.html#terminal-device-operational-modes-mode)

And the resulting [mode-id](https://openconfig.net/projects/models/schemadocs/yangdoc/openconfig-terminal-device.html#terminal-device-operational-modes-mode-mode-id)

Is this the operational-mode you are referring to?

Reply:
Correct thats the operational-mode we are referring to here. We explicitly call out in the test that 

"Verify  that the ZR module Host Interface ID and Media Interface ID
  combination to ZR module AppSel mapping can be configured through the OC
  Operatioanl-Mode."

Available operational modes will be the available AppSel codes.